drivers/sensor/smart_battery/smart_battery.c

```diff
@@ -0,0 +1,284 @@
+/*
+ * Copyright (c) 2021 Leica Geosystems AG
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT smart_battery
+
+#include <drivers/i2c.h>
+#include <init.h>
+#include <drivers/sensor.h>
+#include <sys/__assert.h>
+#include <string.h>
+#include <sys/byteorder.h>
+
+#include "smart_battery.h"
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(smart_battery, CONFIG_SENSOR_LOG_LEVEL);
+
+#define DEV_DATA(dev) ((struct smartbattery_data *const)(dev)->data)
+#define DEV_CFG(dev) \
+	((const struct smartbattery_data *const)(dev)->config)
+
+static int smart_battery_command_reg_read(const struct device *dev, uint8_t reg_addr,
+				    int16_t *val)
+{
+	struct smartbattery_data *smart_battery = DEV_DATA(dev);
```

Comment:
There is no need for DEV_DATA or DEV_CFG macros, just use shorter identifiers

`const struct sbs_config *const cfg = (struct sbs_config *const)(dev->config);`

Reply:
DEV_DATA and DEV_CFG removed and shorter identifiers used